lib/tasks/ember-cli.rake

```diff
@@ -15,4 +15,11 @@ namespace :ember do
   end
 end
 
-task "assets:precompile" => "ember:compile" unless EmberCLI.skip?
+unless EmberCLI.skip?
+  # Hook into assets:precompile:all for Rails 3.1+
+  if Gem::Version.new(Rails::VERSION::STRING) < Gem::Version.new("4.0.0")
```

Comment:
You don't really need `Gem::Version` here. You can just say `Rails::VERSION::MAJOR < 4`.